vignettes/using-based-on.Rmd

```diff
@@ -147,9 +147,11 @@ log_df
 Among other things, the run log contains the tags that have been assigned to each model. Here we use `purrr::map_lgl` to filter the run log to only the final model.
 
 ```{r filtering}
-final_model_path <- log_df %>% 
+final_model_path <- 
+  log_df %>% 
   filter(map_lgl(tags, ~ "final model" %in% .x)) %>%
-  get_model_path()
+  get_model_path() %>% 
```

Comment:
I don't love having this pattern in a vignette. What we want here is the `absolute_model_path` but showing the users that you get that by doing `get_model_path() %>% purrr::map_chr(fs::path_ext_remove)` is a bit silly.

There are a few different ways we could do this, but I think the best is just to use `pull(absolute_model_path)`.